packages/core/admin/ee/admin/pages/SettingsPage/pages/ApplicationInfosPage/components/AdminSeatInfo/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { Flex, Tooltip, Icon, GridItem, Typography, Stack } from '@strapi/design-system';
+import { Link } from '@strapi/design-system/v2/Link';
+import { ExternalLink, ExclamationMarkCircle } from '@strapi/icons';
+import { pxToRem } from '@strapi/helper-plugin';
+import { useLicenseLimits } from '../../../../../../hooks';
+
+const BILLING_STRAPI_CLOUD_URL = 'https://cloud.strapi.io/profile/billing';
+const BILLING_SELF_HOSTED_URL = 'https://share.hsforms.com/1WhxtbTkJSUmfqqEuv4pwuA43qp4';
+
+const AdminSeatInfo = () => {
```

Comment:
I think it would be good to add tests for this component too. Wdyt?

Reply:
I think this one had issues because it needed too many providers?

Reply:
ThemeProvider and IntlProvider - do you see any else?

Reply:
@ivanThePleasant will be able to answer that! 😄

Reply:
I tried, but I have issues with rendering the component for some reason. The errors are weird as well, they don't specify absence of providers, but more the fact that there's an issue rendering the component. I don't think I'll be able to fix this by the time we need to deliver this. Happy to look into it together.

Reply:
I'll have a look and maybe create a PR on top of yours 👍🏼

Reply:
Done in https://github.com/strapi/strapi/pull/15825